L1Trigger/L1TTrackMatch/plugins/L1TrackJetProducer.cc

```diff
@@ -27,8 +29,10 @@
 #include "Geometry/CommonTopologies/interface/PixelGeomDetType.h"
 #include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
 
+#include "DataFormats/L1TCorrelator/interface/TkPrimaryVertex.h"
 #include "DataFormats/L1Trigger/interface/Vertex.h"
```

Comment:
Why add TkPrimaryVertex.h? It should be obsolete wrt Vertex.h

Reply:
It was a relic left when i tried to compare f/w to simulation, and i forgot it. It has been removed now